File: src/main/java/io/scriptor/http/HTTPResultVoid.java
package io.scriptor.http;

import java.io.InputStream;

public class HTTPResultVoid extends HTTPResultBase<Void> {

    public HTTPResultVoid(final int statusCode) {
        super(statusCode);
    }

    public HTTPResultVoid(final int statusCode, final String statusText) {
        super(statusCode, statusText);
    }

    @Override
    public int getSize() {
        return 0;
    }

    @Override
    public InputStream getStream() {
        return null;
    }
}

File: server/src/main/kotlin/dev/scriptor/server/http/result/HTTPResultVoid.kt
package dev.scriptor.server.http.result

class HTTPResultVoid : HTTPResult<Void> {

    constructor(
        statusCode: Int = 200,
        statusText: String = "OK",
        headers: Map<String, String> = emptyMap(),
    ) : super(statusCode, statusText, headers, 0L, 0L, null)
}
